test/workerd/tests.mjs

```diff
@@ -0,0 +1,32 @@
+import assert from "node:assert";
+import process from "node:process";
+
+globalThis.process = process;
+
+// ---- node:crypto ----
+
+export const crypto_getRandomValues = {
+  async test() {
+    const crypto = await import("unenv/runtime/node/crypto");
+
+    const array = new Uint32Array(10);
+    crypto.getRandomValues(array);
+    assert.strictEqual(array.length, 10);
+    assert(array.every((v) => v >= 0 && v <= 0xff_ff_ff_ff));
+  },
+};
+
+// ---- node:url ----
```

Comment:
We have all Node.js upstream tests for `url`, in `test/node/*` now. But since this entry is inlined in capnp config, I'm not sure how to dynamically bring them. If we can, we can reuse same impl.